temporal-sdk/src/main/java/io/temporal/internal/worker/SingleWorkerOptions.java

```diff
@@ -41,7 +41,6 @@ public static Builder newBuilder(SingleWorkerOptions options) {
     private String identity;
     private String binaryChecksum;
     private DataConverter dataConverter;
-    private int taskExecutorThreadPoolSize = 100;
```

Comment:
I think it should stay here instead of poller options

Reply:
Should we pass single worker options to the poller then?